src/rard/research/views/home.py

```diff
@@ -28,3 +29,22 @@ def get_queryset(self):
             .annotate(topic=F("topics__name"))
             .order_by("topic", "topiclink__order")
         )
+
+
+def render_editor_modal_template(request):
+    import re
+
+    from rard.utils.context_processors import symbols_context
+
```

Comment:
have actually gutted this since the way I ended up doing it doesn't require the context to be passed anymore in my testing. Still need the view to generate the url to fetch afaict